### ISSUE TITLE
Game crashes when showing entity info

### ISSUE THREAD

**Issue by redleaf1221** [lovexyn0827/Minecraft-World-Manipulation-Enchantment-Mod#7]
**Describe the bug**
Game crashes when attempting to show the info of a warden that is emerging from the ground.

**To Reproduce**
1. Use sculk shriekers to spawn a warden.
2. F3+E to open entity info hud.
3. Point to the warden emerging from the ground.
4. Game crashes.

**Expected behavior**
The info of warden should be shown correctly.

**Desktop (please complete the following information):**
 - OS: Windows 10 64bit
 - Java Version Java 17.0.7 64bit
 - Minecraft Version 1.20.4
 - Mod Version 0.7.0+v20231216-BETA

**Additional context**
[Video](https://github.com/lovexyn0827/MessMod/assets/68888381/692b6096-2b31-48d7-b58f-06a63b086ca6)
[latest.log](https://github.com/lovexyn0827/MessMod/files/13917716/latest.log)
[crash-2024-01-12_17.57.57-client.txt](https://github.com/lovexyn0827/MessMod/files/13917720/crash-2024-01-12_17.57.57-client.txt)

```
java.lang.NullPointerException: Cannot invoke "Object.toString()" because "v" is null
	at lovexyn0827.mess.rendering.hud.EntityHud.lambda$render$0(EntityHud.java:80)
	at lovexyn0827.mess.rendering.hud.data.LocalDataStorage.lambda$forEach$1(LocalDataStorage.java:42)
	at java.base/java.util.TreeMap.forEach(TreeMap.java:1282)
	at lovexyn0827.mess.rendering.hud.data.LocalDataStorage.forEach(LocalDataStorage.java:41)
	at lovexyn0827.mess.rendering.hud.EntityHud.render(EntityHud.java:77)
	at lovexyn0827.mess.rendering.hud.LookingAtEntityHud.render(LookingAtEntityHud.java:21)
	at lovexyn0827.mess.rendering.hud.ClientHudManager.render(ClientHudManager.java:33)
	at lovexyn0827.mess.MessMod.onRender(MessMod.java:163)
	at net.minecraft.class_310.handler$eca000$messmod$onRender(class_310.java:15787)
	at net.minecraft.class_310.method_1523(class_310.java:1339)
	at net.minecraft.class_310.method_1514(class_310.java:888)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

**Comment by lovexyn0827** [comment]
Fixed in [0.7.1+v20240112](https://modrinth.com/mod/messmod/version/0.7.1+v20240112-BETA)

Thanks for reporting.

**Comment by lovexyn0827** [comment]
By the way, the report is excellent!
